variables.tf

```diff
@@ -274,3 +293,15 @@ variable "primary_metastore_latency" {
   default     = 0
   description = "HMS latency (in ms.) that Waggledance will tolerate.  See \"latency\" parameter in https://github.com/ExpediaGroup/waggle-dance/blob/main/README.md. This sets the latency for the primary metastore only."
 }
+
+variable "waggle_dance_glue_policy" {
+  type        = string
+  default     = ""
+  description = "IAM policy to attach to Waggle Dance Glue IAM role `waggle_dance_k8s_role_iam`."
+}
+
+variable "waggle_dance_glue_assume_role_policy" {
+  type        = string
+  default     = ""
+  description = "K8S assume role policy for Waggle Dance Glue IAM role `waggle_dance_k8s_role_iam`. K8S only"
+}
```

Comment:
Maybe I'm missing something, but why do these two policy docs need to be passed in from the app?  They both seem like something that could be just in the module.

Reply:
@barnharts4 is this ok for you?

Reply:
I would rather we keep the actual IAM policies in the module, and pass in to the module *only* those things that need to change.  

- For `AssumeRole`, that should only be KIAM (or IRSA) role.  Here is an example of how we pass in only the KIAM role ARN to a module: https://github.com/ExpediaGroup/apiary-data-lake/blob/master/iam.tf#L43-L52
- For the `waggle_dance_glue_policy` policy document, this whole document should be in the module, and we should just pass in a variable to the module that is a list of resources that get added to the policy.  Here is an example of how we do something similar for the datalake IAM policies: https://github.com/ExpediaGroup/apiary-data-lake/blob/master/iam-policy-s3-buckets.tf#L92-L95

Reply:
I'm happy to help collaborate on this too.

Reply:
sure no problem, I'm gonna close this PR.